ide/libs.antlr4.runtime/external/binaries-list

```diff
@@ -15,3 +15,6 @@
 # specific language governing permissions and limitations
 # under the License.
 E27D8AB4F984F9D186F54DA984A6AB1CCCAC755E org.antlr:antlr4-runtime:4.7.2
+93058C41A6CBB7F5CA63EDAC837F42C002DBC556 org.antlr:antlr4:4.7.2
+CD9CD41361C155F3AF0F653009DCECB08D8B4AFD org.antlr:antlr-runtime:3.5.2
```

Comment:
Why? This is about antlr4 - so why to 3 runtime?

Reply:
It's strange but the ANTLR4 Tool needs both ANTLR3 and ANTLR4 runtimes.